Lesson 2 - Core JS operators/scripts.js

```diff
@@ -0,0 +1,205 @@
+    b = Math.floor(+prompt('Enter value B'));
+console.log(`A = ${a}`);
+console.log(`B = ${b}`);
+
+if (a >= 0 && b >= 0) {
+   console.log(`A - B = ${a - b}`);
+} else if (a < 0 && b < 0) {
+   console.log(`A * B = ${a * b}`);
+} else if ((a >= 0 && b < 0) || (a < 0 && b >= 0)) {
+   console.log(`A + B = ${a + b}`);
+}
+
+/*
+===============================================================
+Task #4
+===============================================================
+*/
+console.log('TASK #4');
+
+let lowerRange = +prompt('Enter the MIN value for the lower range'),
+    upperRange = +prompt('Enter the MAX value for the lower range'),
+    valueA = Math.floor(Math.random() * (upperRange - lowerRange)) + lowerRange,
+    arrToValueA = [],
+    finalArr = [];
+
+for (let i = lowerRange; i < upperRange + 1; i++){
+   arrToValueA[i] = i;
+}
+
+switch (valueA) {
+   case (valueA):
+      for (let j = valueA; j < arrToValueA.length; j++) {
+         finalArr[j] = j;
+      }
+      alert(finalArr.slice(valueA).join(', '));
+      break;
+}
+//CHECK
+console.log(lowerRange);
+console.log(upperRange);
+console.log(valueA);
+console.log(arrToValueA);
+console.log(finalArr);
+
+/*
+===============================================================
+Task #5
+===============================================================
+*/
+console.log('TASK #5');
+
+let numA = +prompt('Enter a number A'),
+    numB = +prompt('Enter a number B');
+
+// addition
+function getAdd(num1, num2) {
+   return num1 + num2;
+}
+
+// subtraction
+function getSub(num1, num2) {
+   return num1 - num2;
+}
+
+// multiplication
+function getMulti(num1, num2) {
+   return num1 * num2;
+}
+
+// division
+function getDiv(num1, num2) {
+   return Math.floor((num1 / num2) * 100) / 100;
+}
+
+//USED
+console.log(`A = ${numA}`);
+console.log(`B = ${numB}`);
+console.log(`A + B = ${getAdd(numA, numB)}`);
+console.log(`A - B = ${getSub(numA, numB)}`);
+console.log(`A * B = ${getMulti(numA, numB)}`);
+console.log(`A / B = ${getDiv(numA, numB)}`);
+
+/*
+===============================================================
+Task #6
+===============================================================
+*/
+console.log('TASK #6');
+
+let arg1 = Math.floor(Math.random() * 100),
+    arg2 = Math.floor(Math.random() * 100),
+    arrOperation = ['addition', 'subtraction', 'multiplication', 'division'],
+    arrFinalOperation = arrOperation[Math.floor(Math.random() * arrOperation.length)];
+
+//CHECK
+console.log(arg1);
+console.log(arg2);
+console.log(arrOperation);
+console.log(arrFinalOperation);
+
+function mathOperation(arg1, arg2, operation) {
+   switch (operation) {
+      case ('addition'):
+      function getAdd(num1, num2) {
+         return num1 + num2;
+      }
+         console.log(`A + B = ${getAdd(arg1, arg2)}`);
+         break;
+      case ('subtraction'):
+      function getSub(num1, num2) {
+         return num1 - num2;
+      }
+         console.log(`A - B = ${getSub(arg1, arg2)}`);
```

Comment:
Лучше вот так. А console.log уже при использовании
```suggestion
         return getSub(arg1, arg2);
```

Reply:
Поправил.